ale/base/label_pds3.py

```diff
@@ -25,92 +15,202 @@ def label(self):
+        name spicelib expects in bodvrd calls.
 
         Returns
         -------
         : str
-          target name
+          Target name
         """
 
         return self.label['TARGET_NAME']
 
-    @property
-    def _target_id(self):
-        return spice.bodn2c(self.label['TARGET_NAME'])
 
     @property
-    def starting_ephemeris_time(self):
-        if not hasattr(self, '_starting_ephemeris_time'):
-            sclock = self.label['SPACECRAFT_CLOCK_START_COUNT']
-            self._starting_ephemeris_time = spice.scs2e(self.spacecraft_id, sclock)
-        return self._starting_ephemeris_time
+    def sampling_factor(self):
+        """
+        Returns the summing factor from the PDS3 label. For example a return value of 2 
+        indicates that 2 lines and 2 samples (4 pixels) were summed and divided by 4
+        to produce the output pixel value.
 
-    @property
-    def spacecraft_clock_stop_count(self):
-        sc = self.label.get('SPACECRAFT_CLOCK_STOP_COUNT', None)
-        if sc == 'N/A':
-            sc = None
-        return sc
+        Returns
+        -------
+        : int
+          Number of samples and lines combined from the original data to produce a single pixel in this image
+        """
+        return self.label.get('SAMPLING_FACTOR', 1)
 
-    @property
-    def _detector_center_line(self):
-        return spice.gdpool('INS{}_CCD_CENTER'.format(self.ikid), 0, 2)[0]
 
     @property
-    def _detector_center_sample(self):
-        return spice.gdpool('INS{}_CCD_CENTER'.format(self.ikid), 0, 2)[1]
+    def downtrack_summing(self):
+        """
+        Returns the number of detector pixels (normally in the line direction) that
+        have been averaged to produce the output pixel
+
+        Returns
+        -------
+        : int
+          Number of downtrack pixels summed together
+        """
+        return self.label.get('DOWNTRACK_SUMMING', 1)
+
 
     @property
-    def spacecraft_name(self):
+    def crosstrack_summing(self):
         """
-        Spacecraft name used in various Spice calls to acquire
-        ephemeris data.
+        Returns the number of detector pixels (normally in the sample direction) that
+        have been averaged to produce the output pixel
 
+        Returns
+        -------
+        : int
+          Number of crosstrack pixels summed together
+        """
+        return self.label.get('CROSSTRACK_SUMMING', 1)
+
+
+
+    @property
+    def spacecraft_clock_start_count(self):
+        """
         Returns
         -------
         : str
-          Spacecraft name
+          Returns the start clock count string from the PDS3 label.
         """
-        return self.label['MISSION_NAME']
+        return self.label['SPACECRAFT_CLOCK_START_COUNT']
+
 
     @property
-    def detector_line_summing(self):
-        return self.label.get('SAMPLING_FACTOR', 1)
+    def spacecraft_clock_stop_count(self):
```

Comment:
same as start time probably should rename it `clock_start_count`

Reply:
Both of @Kelvinrr comments bring up a point, we agreed to keep the names consistent with what PDS named them, but at what point, if any, do we change a name to help with clarity?

Reply:
If we knew which will be consistent across all labels this would be an easier question to answer.

Reply:
I think for right now we should go with PDS3 unless there's a good reason not to. We can adapt once we get PDS4 sorted.

Reply:
Going with PDS3 names for now, except for start/stop time are now utc_start/stop_time